docs/pages/guides/upgrade.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: guide
+title: Upgrade mesheryctl and Meshery Server
+description: How to upgeade mesheryctl
+permalink: guides/upgrade
+type: guide
+---
```

Comment:
Thank you, @gunjanmimo. 😄 

We're in need of having a section on Meshery as well - 

```
$ mesheryctl upgrade
```

In order to update the Meshery server container images.